js/GroupedBarChart.js

```diff
@@ -174,26 +195,161 @@ var GroupedBarChart = function(param)
               .attr("y", that.h)
               .attr("width", x1.rangeBand())
               .attr("height", 0)
-              .attr("fill", function(d, i) { return z(i); })
+              .attr("fill", function(d, i) { return c(_.indexOf(data, d)); })
               // The transition produces a bouncing effect for the bar
               .transition()
                 // Each bar will be delayed depending on its position in the graph
                 .delay(function(d, i) { return  _.indexOf(data, d) * 50;} )
-                .duration(150)
+                .duration(barsAnimationTime/3)
                 // Expand height first (bounce effect)
                 .attr('y', function(d) { return y(d.yval) - 50; })
-                .attr('height', function(d) { return (that.h - y(d.yval)) + 50 })
+                .attr('height', function(d) { return (that.h - y(d.yval)) + 50 ;})
                   .transition()
-                  .duration(150)
+                  .duration(barsAnimationTime/3)
                   // Lower the height after (bounce effect)
                   .attr('y', function(d) { return y(d.yval) + 15; })
                   .attr('height', function(d) { return that.h - y(d.yval) - 15; })
                     .transition()
-                    .duration(150)
+                    .duration(barsAnimationTime/3)
                     // Turn back to original height
                     .attr('y', function(d) { return y(d.yval); })
-                    .attr('height', function(d) { return that.h - y(d.yval); })
+                    .attr('height', function(d) { return that.h - y(d.yval); });
+
+
+                    /*Data point funcitons*/
+                    var c_mouseover = function(d){
```

Comment:
The mouse overs were done in this style in the tornado chart unless you changed it. It was because Internet explore didn't like functions within functions